### PR TITLE
[MNT] replace author names by GitHub ID in author fields, linting

### PR DESCRIPTION
**Pull request by fkiraly**
This PR replaces some author names by GitHub IDs in author fields.

This is the convention, and full names are still linked via the contributor register.

Also lints all the affected files - no content is changed.

### CODE REVIEW

File: sktime/utils/validation/series.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 """Functions for checking input data."""
 
-__author__ = ["Markus Löning", "Drishti Bhasin", "khrapovs"]
+__author__ = ["mloning", "Drishti Bhasin", "khrapovs"]
```
Comment (aiwalter) [inline sktime/utils/validation/series.py:6]: what about `Drishti Bhasin`?

File: sktime/utils/validation/__init__.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "check_n_jobs",
     "check_window_length",
 ]
-__author__ = ["Markus Löning", "Taiwo Owoseni", "khrapovs"]
+__author__ = ["mloning", "Taiwo Owoseni", "khrapovs"]
```
Comment (aiwalter) [inline sktime/utils/validation/__init__.py:15]: what about `Taiwo Owoseni`?

File: sktime/transformations/panel/tsfresh.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 """tsfresh interface class."""
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 
-__author__ = ["AyushmaanSeth", "Markus Löning", "Alwin Wang"]
+__author__ = ["AyushmaanSeth", "mloning", "Alwin Wang"]
```
Comment (aiwalter) [inline sktime/transformations/panel/tsfresh.py:5]: what about `Alwin Wang`?

File: sktime/utils/validation/series.py (hunk 1 above)
Comment (fkiraly) [reply sktime/utils/validation/series.py:6]: don't know what their github name is

File: sktime/utils/validation/__init__.py (hunk 2 above)
Comment (fkiraly) [reply sktime/utils/validation/__init__.py:15]: not caught by bulk-replace. Someone else can do that and do all the linting ;-)

File: sktime/transformations/panel/tsfresh.py (hunk 3 above)
Comment (fkiraly) [reply sktime/transformations/panel/tsfresh.py:5]: don't know github id

**Review by aiwalter — APPROVED** [review]

**Pull request merged**